src/features/Header/Categories/Categories.module.scss

```diff
@@ -18,8 +17,11 @@
   transition: transform 300ms ease-in-out;
 
   &[data-open='true'] {
```

Comment:
Please use `aria-expanded`. It will do the same thing but it’s better for the accessibility